extra/hhvm.conf

```diff
@@ -24,3 +24,9 @@ hhvm.repo.central.path = /var/run/hhvm/hhvm.hhbc
 hhvm.mysql.socket = /var/run/mysqld/mysqld.sock
 hhvm.pdo_mysql.socket = /var/run/mysqld/mysqld.sock
 hhvm.mysqli.socket = /var/run/mysqld/mysqld.sock
+
+# MySQL Optimization
+# Use persistent connections and higher timeout
```

Comment:
Nit:  You should remove the no-longer-accurate comment about _persistent connections_.